lib/zstd.h

```diff
@@ -600,7 +602,7 @@ ZSTDLIB_API const ZSTD_DDict* ZSTD_initStaticDDict(
 typedef void* (*ZSTD_allocFunction) (void* opaque, size_t size);
 typedef void  (*ZSTD_freeFunction) (void* opaque, void* address);
 typedef struct { ZSTD_allocFunction customAlloc; ZSTD_freeFunction customFree; void* opaque; } ZSTD_customMem;
-static ZSTD_customMem const ZSTD_defaultCMem = { NULL, NULL, NULL };  /**< this constant defers to stdlib's functions */
+#define ZSTD_defaultCMem ((ZSTD_customMem){ NULL, NULL, NULL})   /**< this defers memory management to stdlib's functions */
```

Comment:
In most cases, `ZSTD_defaultCMem` is used directly as function argument.
It would be quite a change to force the creation of a local variable just to store `{NULL, NULL, NULL}` into it.